src/components/library/LibrarySort.js

```diff
@@ -12,9 +12,10 @@ import { withStyles } from "@material-ui/core/styles";
 const sorts = [
   { flagState: "ALPHA", description: "Alphabetically" },
   // "LAST_READ" and "LAST_UPDATED" not yet implemented
-  { flagState: "UNREAD", description: "Unread" },
-  { flagState: "TOTAL", description: "Total chapters" },
-  { flagState: "SOURCE", description: "Source" }
+  { flagState: "LAST_UPDATED", description: "Last updated" },
+  { flagState: "UNREAD", description: "Unread" }
+  // { flagState: "TOTAL", description: "Total chapters" },
```

Comment:
Why have these sorts been commented out?

Reply:
The total chapters are no longer supplied as a field in the manga object for performance reasons. The tachiyomi app works around this by only loading the total chapters when the `TOTAL` sort option is selected. I'm currently thinking about how I can architect this so we can still support the `TOTAL` sort option while keeping performance good for large libraries.

The `SOURCE` sort option is supposed to sort manga by their source names. But with the v3 API, the source name is no longer provided with the manga object, instead the source id provided. So I have to fetch the sources before we have enough information to use this sort.